src/content/en/2019/media.md

```diff
@@ -8,316 +7,276 @@ authors: [colinbendell, dougsillars]
 discuss: 1759
 reviewers: [ahmadawais, eeeps]
 published: 2019-11-11T00:00:00.000Z
-last_updated: 2019-11-15T00:00:00.000Z
+last_updated: 2019-11-16T00:00:00.000Z
 ---
 
 ## Introduction
 Images, animations, and videos are an important part of the web experience. They are important for many reasons - they help tell stories, engage audiences and provide artistic expression in ways that often cannot be easily produced with other web technologies. The importance of these media resources can be demonstrated in two ways: by the sheer volume of bytes required to download for a page, and also the volume of pixels painted with media. 
 
-From a pure bytes perspective, HTTP Archive has historically reported an average of two-thirds of resource bytes associated from media. From a distribution perspective, we can see that virtually every web page depends on images and videos. Even at the 10th percentile, we see that 43% of the bytes are from media and can rise to 91.3% of the total bytes at the 90th percentile of pages. 
+From a pure bytes perspective, HTTP Archive has historically reported an average of two-thirds of resource bytes associated from media. From a distribution perspective, we can see that virtually every web page depends on images and videos. Even at the tenth percentile, we see that 43% of the bytes are from media and can rise to 91.3% of the total bytes at the 90th percentile of pages. 
 
 <figure>
-  <iframe aria-labelledby="fig1-caption" width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=1189524305&format=interactive"></iframe>
-  <a href="/static/images/2019/04_Media/fig1.png" class="fig-mobile">
-    <img src="/static/images/2019/04_Media/fig1.png" aria-labelledby="fig1-caption" width="600">
+  <iframe title="Figure 1. Web page bytes: image and video versus other." aria-labelledby="fig1-caption" aria-describedby="fig1-description" width="600" height="371" seamless frameborder="0" scrolling="no" src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSViHIntdF6-bHAI0cl1HelY_X8rR4lf0P3W2Y8I5SyVMxG-ptggTHfWA0qrrU47RvuAydLE6Zex6L3/pubchart?oid=1189524305&format=interactive"></iframe>
```

Comment:
Is `title` necessary?

Do we need both `labelledby` and `describedby`?

Reply:
It fails Firefox’s accessibility test without it. Title kind of acts like an alt tag for iframes. In my auto generation script as part of https://github.com/HTTPArchive/almanac.httparchive.org/pull/491 I set the title to the img alt text automatically.

Here was my thinking:

- alt/title - there for SEO/validation purposes. Can be different but currently just set to figcaption in most places.
- labelledby - there to link figcaption to prevent screen readers reading same text twice (once for alt, once for figcaption). Overrides alt/title is this takes priority for screen readers compared to those.
- describedby - deeper description of the image than just the alt/label/caption.

WDYT?

Reply:
It probably can't hurt to be _more_ accessible but I was just thinking that we should keep all data viz markup consistent and if/when we optimize it we can do everything at once. I'd also like to hear from one of the a11y experts whether these redundancies are helpful.

These changes are ok for now if you're sure they make the experience better. We should revisit this in #491 or in a follow-up issue.

Reply:
Not sure at all - like a lot of a11y it does seem there's a bit of guess work, but been reading up on it and do think this is the right way to use them. Would be nice if an a11y expert could give their input. Plus SOE and a11y requirements seem to clash sometimes (don't have an alt tag/complain alt-tag is missing) 🤨

Also I blame you for having such detailed descriptions in the chapters you edited - on top of fig captions! 😉

I already have a branch with the other chapters ready to go: https://github.com/HTTPArchive/almanac.httparchive.org/tree/figures_new_format/src/content/en/2019 but I want to push #491 to prod first and check it definitely fixes the issues it's supposed to on a few devices with the two example chapters, before doing the other chapters so will do a follow up PR after that testing is complete.